### PR TITLE
Update UUIDs of Services

### PR DESCRIPTION
**Pull request by openBackhaul**
**Process for updating the UUIDs of Services**
- [x] Thorsten to check categories of OperationServers and OperationClients in ServiceList
- [x] Iswaryaa to review categories of OperationServers and OperationClients in ServiceList
- [x] Katharina to add new UUIDs based on category of operation (but not old UUID) in ServiceList
- [x] Iswaryaa to review new UUIDs in ServiceList
- [x] Katharina to CTRL+h for old UUIDs and replace them by new UUIDs in ForwardingList, CONFIGfile and OAS.
- [x] Katharina to step through ForwardingList, CONFIGfile and OAS and compare every UUID against new one in ServiceList
- [ ] Iswaryaa to review new UUIDs in ForwardingList, CONFIGfile and OAS
If no errors found
- [ ] Iswaryaa to delete old UUIDs in ServiceList
- [ ] Iswaryaa to merge the feature branch into develop
- [ ] Iswaryaa to close this issue and delete the feature branch on GitHub

### CODE REVIEW

**Comment by IswaryaaS** [comment]
OamLog+services.yaml file looks fine for categories of services

File: OamLog+services.yaml [hunk 1]
```diff
@@ -90,16 +119,19 @@ clients:
...
       tcp-port: 1003
       uuid: ol-1-0-0-tcp-c-0010
+            ol-1-0-0-tcp-c-im-nr-1-0-0-000
     operation-clients:
       own-oam:
         basic:
         individual:
           - service-name: /v1/regard-application
             uuid: ol-1-0-0-op-c-1010
+                  ol-1-0-0-op-c-im-ne-1-0-0-000
```
Comment (IswaryaaS) [inline OamLog+services.yaml:134]: A small typo mistake here in uuid ol-1-0-0-op-c-im-ne-1-0-0-000 . application-id to be replaced with nr instead of ne

**Review by IswaryaaS — CHANGES_REQUESTED** [review]
@kmohr-soprasteria , requesting change for a single comment

**Review by IswaryaaS — APPROVED** [review]
Changes in service.yaml file looks fine. We can now proceed for updating other files.

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

**Comment by kmohr-soprasteria** [comment]
- Talked to Thorsten about the fc uuids - those shall not be changed with this pull request.
- Merged changes from develop branch
- @IswaryaaS: you can continue with your part

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**